.github/workflows/agent-build.yml

```
@@ -1,37 +1,52 @@
+        description: "Build Version"
         required: true
+
+permissions:
+  id-token: write
+  contents: write
+
 jobs:
   Build:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)